### PR TITLE
Completed css-grid-classes.

### PR DESCRIPTION
**Pull request by Youssef-Najjarine**
image

### CODE REVIEW

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Hey Youssef, decent initial implementation on this. But there are a few things which need to be adjusted:
1. The exercise is supposed to be completed with grid classes as described in the instructions. 
2. Grid classes means using a container element, and a series of rows with columns that are percentage based. 
3. Please refactor this exercise with that in mind, if you have questions as you are working on it, please reach out!

File: css-grid-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <header><h1>Choose your character!</h1></header>
+      </div>
+    </div>
+    <div class="row1">
+      <div class="column-one-third">
```
Comment (BrettAlbright) [inline css-grid-classes/index.html]: you want to have 3 `column-one-third` classes in this row, one for link, one for the text, and one for yoshi and each should be 1/3 width of the whole row in percentage

File: css-grid-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+          existing Smash gameplay elements,<br>
+          although its online mode was widely<br>
+          criticized.
+        </p>
+        </div>
+        <aside><img src="images/yoshi.png"/></aside>
+      </div>
+
+
+          <div class="row2">
```
Comment (BrettAlbright) [inline css-grid-classes/index.html]: There should only be a single `row` class that is used for each row

File: css-grid-classes/index.html [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+            <div class="column-one"><img src="images/kirby.png"/></div>
```
Comment (BrettAlbright) [inline css-grid-classes/index.html]: each of these elements should have the same column class set to be 1/4 the total width as a percentage.

File: css-grid-classes/index.html [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+            <div class="column-two"><img src="images/fox.png"/></div>
+            <div class="column-three"><img src="images/pikachu.png"/></div>
+            <div class="column-four"><img src="images/ness.png"/></div>
+          </div>
+          <div class="row3">
```
Comment (BrettAlbright) [inline css-grid-classes/index.html]: should be the single row class

File: css-grid-classes/index.html [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+            <div class="column-one">
```
Comment (BrettAlbright) [inline css-grid-classes/index.html]: should be two columns in this row, one that is 1/3 the length for the text, and one that is 2/3 the length for the image

File: css-grid-classes/index.html [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+                puzzle types, featuring Donkey Kong as the opponent in an industrial<br>
+                construction setting.
+              </p>
+            </div>
+            <aside><img  src="images/dk.png"/></aside>
+          </div>
+
+
+    </div>
+    <div class="row4">
```
Comment (BrettAlbright) [inline css-grid-classes/index.html]: single row class

File: css-grid-classes/index.html [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+
+      <div class="column-one">
```
Comment (BrettAlbright) [inline css-grid-classes/index.html]: these column widths should be the same wisth as the row above

File: css-grid-classes/styles.css [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+ padding: 40px;
+}
+.section4 {
+  margin-top: 50px;
+} */
+
+body {
+  margin-left: 100px;
+}
+.column-one-third,
```
Comment (BrettAlbright) [inline css-grid-classes/styles.css]: remember, there should be only one `row` class set to display flex. And all of the column classes need to be given proper width values.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Moving in the right direction! A few more updates are noted.

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**